zerver/lib/dev_ldap_directory.py

```diff
@@ -25,7 +25,7 @@ def generate_dev_ldap_dir(mode: str, num_users: int = 8) -> Dict[str, Dict[str,
         ldap_data.append((name, email, phone_number, birthdate))
 
     profile_images = []
-    for path in glob.glob(os.path.join(static_path("images/landing-page/team"), "*")):
+    for path in glob.glob(os.path.join(static_path("images/people"), "*")):
```

Comment:
I think for this one we probably just want to switch to using some photos of people that are licensed to be used for any purpose in the tests, and leave the photo of me for the landing pages; @alya I think has a source for such images.

I think they could then go in `zerver/tests/fixtures/avatars/`.

Reply:
A couple of profile photos based off Pexels (https://www.pexels.com/license/):

![pexels-justin-shaifer-1222271](https://user-images.githubusercontent.com/2090066/198122745-a8dfbe19-44c7-44db-84cd-cb652b187179.jpg)
![pexels-daniel-xavier-1239288](https://user-images.githubusercontent.com/2090066/198122750-43390cdf-d79f-46f5-9d23-b122a39cffdd.jpg)

Reply:
I added these two images in a prep commit to `static/images/test-images/avatars/` and then followed with a commit to use them instead of `tim.png`. I didn't include any information about where the images come from in the commit message, because I wasn't sure of the best wording.